spec/models/allocation_spec.rb

```diff
@@ -128,4 +128,21 @@
       end
     end
   end
+
+  describe '#hourly_rate_currencies' do
+    let!(:brl_allocations) { create_list(:allocation, 2, hourly_rate_currency: 'BRL') }
+    let!(:usd_allocation) { create(:allocation, hourly_rate_currency: 'USD') }
+
+    subject { described_class.hourly_rate_currencies }
+
+
```

Comment:
Unnecessary newline 💅

Reply:
When it comes to specs/examples structure i get confused sometimes, isn't it better to explicit the subject on each context/describe? By doing this it gets more confusing for those reading the test? Or does it gets easier?